### PR TITLE
SOL-1480 Allowance Section UI changes

### PR DESCRIPTION
**Pull request by afzaledx**

### CODE REVIEW

**Comment by douglashall** [comment]
@afzaledx I tried this out and it looks like it is still a work in progress. Let's see where we are at with this tomorrow.

**Comment by afzaledx** [comment]
@douglashall, kind of late now. Hasnain and I pushed a few Jasmine tests but did not proceed with bumping the version. Will do that tomorrow.

File: edx_proctoring/static/proctoring/js/views/proctored_exam_add_allowance_view.js [hunk 1]
```diff
@@ -155,6 +162,47 @@ var edx = edx || {};
...
+            }
+            else {
+                $('#exam_type_label').text(gettext('Timed Exam'));
+
+                // Even though we have the Select element hidden, the backend will still be using
+                // the Select's value for the allowance type (key).
+                $('#allowance_type').val('additional_time_granted').hide();
+
+                $('#timed_exam_allowance_type, #timed_exam_mins_label').show();
+                $('#allowance_value_label').text(gettext('Additional Time'));
```
Comment (douglashall) [inline edx_proctoring/static/proctoring/js/views/proctored_exam_add_allowance_view.js]: Can we replace the above two lines with the following?
```
this.selectAllowanceLabelsText('additional_time_granted');
```
And move the '#timed_exam_allowance_type` toggle into the `selectAllowanceLabelsText` function?

Comment (afzaledx) [inline edx_proctoring/static/proctoring/js/views/proctored_exam_add_allowance_view.js]: @douglashall, refactored some code. Can you please look at it again?

**Comment by douglashall** [comment]
:+1:

**Pull request merged**